pkg/application/application.go

```diff
@@ -142,20 +135,17 @@ func GetCurrent(client *occlient.Client) (string, error) {
 // Do not use for read operations like get, list; only for write operations like
 // create
 func GetCurrentOrGetCreateSetDefault(client *occlient.Client) (string, error) {
-	currentApp, err := GetCurrent(client)
+	projectName := project.GetCurrent(client)
+	currentApp, err := GetCurrent(projectName)
 	if err != nil {
 		return "", errors.Wrap(err, "unable to get active application")
 	}
-
 	// if no Application is active use default
 	if currentApp == "" {
 		// get default application name
 		currentApp = getDefaultAppName()
 		// create if default application does not exist
-		exists, err := Exists(client, currentApp)
-		if err != nil {
-			return "", errors.Wrapf(err, "unable to check if app %v exists", currentApp)
-		}
+		exists, _ := Exists(client, currentApp)
```

Comment:
Why are we ignoring the error here too? Shouldn't we exit if there is an error here?

Reply:
we just need to check whether app exist or not, depend on that we are creating the application, that's why we don't care about error here